core/domain/classifier_services.py

```diff
@@ -118,3 +122,68 @@ def classify_string_classifier_rule(state, normalized_answer):
             return None
 
     return None
+
+
+def get_classifier_from_model(classifier_model):
+    """Returns a classifier domain object given a classifier model loaded
```

Comment:
Ah -- take a look at docstrings written in files as part of #2394. There is a standard format for this (where definitions of args etc. are specified), and this file should follow it.